lib/components/viewers/route-viewer.js

```diff
@@ -159,11 +164,13 @@ class RouteRow extends PureComponent {
           onClick={this._onClick}
         >
           <div style={{display: 'inline-block'}}>
-            {// TODO: re-implement multi-agency logos for route viewer.
-              // Currently, the agency object is not nested within the get all
-              // routes endpoint and causing this to only display transitOperators for
-              // the selected route.
-              // operator && <img src={operator.logo} style={{marginRight: '5px'}} height={25} />
+            {operator &&
```

Comment:
Perhaps this check for operator existing should extend to the logo field being defined as well.